charts/kubernetes-zfs-provisioner/templates/storageclass.yaml

```
@@ -19,5 +19,6 @@ parameters:
   type: {{ .type | default "nfs" }}
   node: {{ .node | default "''" }}
   shareProperties: {{ .shareProperties | default "''" }}
+  reserveSpace: {{ .reserveSpace || "true" }}
```

That looks invalid. I think you're missing the default function
```suggestion
  reserveSpace: {{ .reserveSpace | default "true" }}
```

---

Yep, it actually doesn't run, see the job: https://github.com/ccremer/kubernetes-zfs-provisioner/actions/runs/5824298579/job/15810294131?pr=103

I wonder why the command didn't exit non-zero though .... :thinking:

---

Yep, I think I looked at the other code and then my brain turned that into something more familiar/js-like. I vaguely remember running helm lint, but I didn't tick the box that I ran `make chart-docs`... oh well.